dbt/adapters/bigquery/jobs.py

```diff
@@ -0,0 +1,3 @@
+def define_job_id(model_name, invocation_id):
+    job_id = f"{model_name}_{invocation_id}"
```

Comment:
What's the constraint on `job_id`? Is there a max length? Can all characters that go into a model name also be used in a job id?

Reply:
will definitely have to test this, I think all characters are fine as we should just be combining 2 strings but length may hit a limit

Reply:
I would want to make sure that we can submit a job_id to BQ with some weird characters. People put all kinds of things in their model names. An alternative is to hash the model_name so that it's only alpha-numeric.

Reply:
+1 if we want to stick with uuid we can just generate a deterministic one with `uuid.uuid5`